app/scripts/controllers/mobileClients.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:MobileClientsController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('MobileClientsController',
+    function ($scope,
+              $filter,
+              $routeParams,
+              Constants,
+              DataService,
+              ProjectsService) {
+
+      $scope.projectName = $routeParams.project;
+      $scope.emptyMessage = "Loading...";
+      $scope.alerts = {};
+      $scope.imagesByDockerReference = {};
+      $scope.redirectUrl = "/project/" + $scope.projectName + "/overview";
+      $scope.breadcrumbs = [
```

Comment:
Does all this work by attaching to the controller? Might be able to get rid of the `$scope` import